release-notes.txt

```diff
@@ -1,3 +1,540 @@
+RIOT-2020.10 - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release:
+===================
+
+The 2020.10 release includes:
+
+ - Support for PicoLIBC as standard C library implementation
+ - A new radio abstraction layer for ieee802.15.4 radios
+ - Improvement to the linking of modules
+ - An improved algorithm for generating local unique identifiers
+ - Pagewise addressing support for MTD devices
+ - 23 additional modules supported by Kconfig
+ - Initial rework of the clock modelling on stm32 devices
+ - 5 new boards, 2 new device drivers, 7 packages updated
+
+48 pull requests, composed of 1355 commits, have been merged since the
```

Comment:
only 48 PRs, are you sure ? With 1355 commits that a mean of ~28 commits per PR. There's something wrong here :)

Reply:
Its 482 on my count, so @bergzand might just have forgotten a digit ;-)

- [465 merged to `master` between `2020.10-devel` and `2021.01-devel`](https://github.com/RIOT-OS/RIOT/pulls?q=is%3Apr+merged%3A2020-07-10T15%3A35%3A10..2020-10-04T13%3A04%3A31+base%3Amaster+is%3Amerged)
- [17 merged to `2020.10-branch`](https://github.com/RIOT-OS/RIOT/pulls?q=is%3Apr++base%3A2020.10-branch+is%3Amerged)

Reply:
Yup, it's wrong, it should be 484 PRs.

Reply:
> Its 482 on my count, so @bergzand might just have forgotten a digit ;-)

I count ~~484~~ 485 merge commits between 2020.07.1 and 2020.10-RC4. Maybe your number is technically more correct though :)

Reply:
Anyway, fixed in the latest push!

Reply:
Between 2020.07.1? That includes the backports to `2020.07` I believe... though that 3 seems small.

Reply:
I also get 485 when checking in the terminal...

```console
$ git log --merges --oneline 2020.10-devel..2020.10-RC4 | wc -l
485
```

Don't want to check now why ^^